apps/gitness/src/pages/repo/types/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum orderSortDate {
```

Comment:
Move instead to `app/src/gitness/types.ts`.

Reply:
fixed